src/ATen/ExpandUtils.h

```diff
@@ -5,6 +5,9 @@
 
 namespace at {
 
+std::vector<int64_t> infer_size2(IntList a, IntList b);
```

Comment:
Might be worth adding a comment here what the `2` means - it looks a little confusing without its friends `infer_size3` or whatever it was.

Reply:
err, I can just rename it to "infer_size" -- we are in C++, so we don't need unique names.